### PR TITLE
Update vim packages

### PR DESCRIPTION
**Pull request by uberhacker**
Tested on all architectures.

### CODE REVIEW

File: packages/gvim.rb [hunk 1]
```diff
@@ -71,5 +71,15 @@ def self.postinstall
...
     puts
+    puts "If you receive the following error while attempting to launch gvim:".orange
+    puts
+    puts "E233: cannot open display"
+    puts "Press ENTER or type command to continue"
+    puts "E852: The child process failed to start the GUI"
+    puts "Press ENTER or type command to continue"
+    puts
+    puts "Execute the following:".orange
+    puts "echo 'export GDK_BACKEND=x11' >> ~/.bashrc && source ~/.bashrc".orange
```
Comment (JL2210) [inline packages/gvim.rb]: Perhaps we should change this in `sommelier` instead of overwriting the variable in every package that it breaks?

Comment (uberhacker) [reply packages/gvim.rb]: That is a good thought.  This fix is needed for code/codium also.

**Comment by skiman6010** [comment]
Do you want to rework this before I merge it or should I just go ahead and merge it and you can rework it when you rework sommelier?

**Comment by uberhacker** [comment]
Hold off.  I will update to a newer version after work this evening.

**Pull request closed without merging**